wiki/Tournaments/oTTMT/1/en.md

```diff
@@ -0,0 +1,285 @@
+---
+tags:
+- oTTMT
+---
+# osu! Tag Team Multiplayer Tournament
+
+![oTTMT Logo](img/logo.png)
+
+The **osu! Tag Team Multiplayer Tournament** (***oTTMT***) was a two-on-two tag team based osu! tournament hosted by ![][flag_SE] [Gabi](/users/57057). It was the 1st installment of the osu! Tag Team Multiplayer Tournament.
+
+## Tournament Schedule
+
+- The osu! Tag Team Multiplayer Tournament started 2009-06-12 and ended 2009-09-06
+
+## Prizes
+
+In every tournament, people conquering a place on the podium are eligible to receive unique prizes created specifically for osu! Tag Team Multiplayer Tournament winners. It is possible that these items change with every installment of the osu! Tag Team Multiplayer Tournament.
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 6 months of osu!supporter tag, profile badge |
```

Comment:
add image of the profile badge somewhere around here (seems like other tournament articles add it below this table?)